src/main/java/org/aion/wallet/util/SyncStatusFormatter.java

```diff
@@ -0,0 +1,42 @@
+package org.aion.wallet.util;
+
+import org.aion.api.server.types.SyncInfo;
+
+public class SyncStatusFormatter {
+
+    public static final int SECONDS_IN_A_MINUTE = 60;
+    public static final int SECONDS_IN_A_HOUR = 3600;
+    public static final int SECONDS_IN_A_DAY = 86400;
+    public static final String UNDEFINED = "Undefined";
+    public static final String UP_TO_DATE = "Up to date";
+
+    public static String formatSyncStatus(SyncInfo syncInfo) {
```

Comment:
as mentioned above, we should move away from core dependecies and use out own objects